includes/class-amp-theme-support.php

```diff
@@ -19,6 +19,27 @@ class AMP_Theme_Support {
 	 */
 	const SCRIPTS_PLACEHOLDER = '<!-- AMP:SCRIPTS_PLACEHOLDER -->';
 
+	/**
+	 * Response cache group name.
+	 *
+	 * @var string
+	 */
+	const RESPONSE_CACHE_GROUP = 'amp-reponse';
+
+	/**
+	 * Query var that triggers response cache removal for the given page.
+	 *
+	 * @var string
+	 */
+	const FLUSH_RESPONSE_CACHE_VAR = 'amp_flush_response_cache';
+
+	/**
+	 * Response cache hash key.
+	 *
+	 * @var string
+	 */
+	public static $response_cache_key;
```

Comment:
Since this variable is only ever used inside of the `prepare_response` method, I don't think it needs a class variable.

Reply:
It was used in `prepare_response()`, `set_response_cache_key()` and in the tests but it is no longer necessary since `set_response_cache_key()` was removed in favour of `wp_json_encode()` [in this commit](https://github.com/Automattic/amp-wp/pull/1156/commits/9bcc4acfb28d87ba04b5f23c3793cce83b5deaa3).